java-client/src/test/java/org/opensearch/client/opensearch/integTest/AbstractCatClientIT.java

```diff
@@ -242,7 +242,7 @@ public void testCatPitSegments() throws Exception {
 
         assertNotNull("PitSegmentsResponse.segments() is null", PitSegmentsResponse.valueBody());
         assertTrue("PitSegmentsResponse.segments().size() == 0",
-                PitSegmentsResponse.valueBody().size() > 0);
+                PitSegmentsResponse.valueBody().isEmpty());
```

Comment:
Finally, tests becomes logical: it was an issue fixed recently, the Cat API for PIT segments returned header row twice.